catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -246,8 +246,11 @@ public IcebergSchema alterSchema(NameIdentifier ident, SchemaChange... changes)
       List<String> removals = new ArrayList<>();
       Map<String, String> updates = new HashMap<>();
       Map<String, String> resultProperties = new HashMap<>(metadata);
+      String comment = null;
       for (SchemaChange change : changes) {
-        if (change instanceof SchemaChange.SetProperty) {
+        if (change instanceof SchemaChange.UpdateComment) {
+          comment = ((SchemaChange.UpdateComment) change).getNewComment();
```

Comment:
I think you we should store `comment` in  Iceberg schema properties like hive. or we couldn't load the latest comment in `loadSchema`

Reply:
@Clearvive  please  check the Iceberg related logics.